src/modules/roc_netio/target_libuv/roc_netio/udp_sender_port.cpp

```diff
@@ -252,13 +262,10 @@ packet::PacketPtr UdpSenderPort::read_() {
 
 void UdpSenderPort::close_() {
     if (closed_) {
-        return; // handle_closed() was already called
+        return;
```

Comment:
NITPICK: let's rearrange the check like we have in UdpReceiverPort:

```
// check if was successfully opened
// check if was closed
// close
```

Reply:
Done.